static/app/views/onboarding/otherSetup.tsx

```diff
@@ -0,0 +1,128 @@
+import {PlatformKey} from 'app/data/platformCategories';
+import {t, tct} from 'app/locale';
+import {Organization, Project} from 'app/types';
+import {analytics} from 'app/utils/analytics';
+import getDynamicText from 'app/utils/getDynamicText';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import {ProjectKey} from 'app/views/settings/project/projectKeys/types';
+
+import FirstEventFooter from './components/firstEventFooter';
+import FullIntroduction from './components/fullIntroduction';
+import {StepProps} from './types';
+
+type AnalyticsOpts = {
+  organization: Organization;
+  project: Project | null;
+  platform: PlatformKey | null;
+};
+
+const recordAnalyticsDocsClicked = ({organization, project, platform}: AnalyticsOpts) =>
+  analytics('onboarding_v2.full_docs_clicked', {
+    org_id: organization.id,
+    project: project?.slug,
+    platform,
+  });
+
+type Props = StepProps & {
+  api: Client;
+  organization: Organization;
+} & AsyncComponent['props'];
+
+type State = {
+  keyList: Array<ProjectKey> | null;
+} & AsyncComponent['state'];
+
+class OtherSetup extends AsyncComponent<Props, State> {
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      keyList: null,
+    };
+  }
+
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, project} = this.props;
+    return [['keyList', `/projects/${organization.slug}/${project?.slug}/keys/`]];
+  }
+
+  handleFullDocsClick = () => {
+    const {organization, project, platform} = this.props;
+    recordAnalyticsDocsClicked({organization, project, platform});
+  };
+
+  render() {
+    const {organization, project} = this.props;
+    const {keyList} = this.state;
+
+    const currentPlatform = 'other';
+
+    const introduction = <FullIntroduction currentPlatform={currentPlatform} />;
+
+    const blurb = (
+      <React.Fragment>
+        <p>
+          {tct(`Prepare the SDK for your language following this [docsLink:guide].`, {
+            docsLink: <ExternalLink href="https://develop.sentry.dev/sdk/overview/" />,
+          })}
+        </p>
+
+        <p>
+          {t('Once your SDK is set up, use the following DSN and send your first event!')}
+        </p>
+
+        <p>{tct('Here is the DSN: [DSN]', {DSN: <b> {keyList?.[0].dsn.public}</b>})}</p>
+      </React.Fragment>
+    );
+
+    const docs = (
+      <DocsWrapper>
+        {blurb}
+        {project && (
+          <FirstEventFooter
+            project={project}
+            organization={organization}
+            docsLink="https://develop.sentry.dev/sdk"
+            docsOnClick={this.handleFullDocsClick}
+          />
+        )}
+      </DocsWrapper>
+    );
+
+    const testOnlyAlert = (
+      <Alert type="warning">
+        Platform documentation is not rendered in for tests in CI
+      </Alert>
+    );
+
+    return (
+      <React.Fragment>
+        {introduction}
```

Comment:
@roggenkemper No need to make a variable out of `introduction` then use it. You can just instatiate the component in here directly:
```
<React.Fragment>
   <FullIntroduction currentPlatform={currentPlatform}
```

Reply:
just fixed this!